### ISSUE TITLE
Please dont import the whole rxjs library

### ISSUE THREAD

**Issue by ruimserrano** [sureshchahal/angular2-adal#38]
Hi,

I notice that you import observer from rxjs in adal.service.ts and authHttp.service.ts
```
import {Observable} from 'rxjs';
```

but you should import like this
```
import { Observable } from 'rxjs/Observable';
```

so you dont import the whole library (+200kb)

thanks

**Comment by ruimserrano** [comment]
i've cloned, created a branch and changed the code.
Tested on my project, was able to shrink my vendor bundle in 487kb less.
Your lib is including all rxjs, but only needs mergeMap, catch,map and bindCallBack.

I dont have permissions to push my branch to issue a pull request.
Do you want to see it and evaluate? 

Cheers

**Comment by sureshchahal** [comment]
changing just to 'rxjs/Observable' throw error error TS2339: Property 'bindCallback' does                                                                                                                                                  not exist on type 'typeof Observable'.

is there some other change required to make it work? Also if you can submit pull request, I can check and publish

**Comment by slubowsky** [comment]
import 'rxjs/add/observable/bindCallback';

**Comment by gert-janvercauteren** [comment]
I believe #48 implements the changes you suggest

**Comment by sureshchahal** [comment]
fixed with #48
